.eslintrc.json

```diff
@@ -221,6 +221,10 @@
     "@typescript-eslint/no-shadow": "error",
     "@typescript-eslint/no-meaningless-void-operator": "error",
     "no-use-before-define": "off",
+    "@typescript-eslint/no-misused-promises": [
+      "error", // https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-misused-promises.md#checksvoidreturn
+      { "checksVoidReturn": { "attributes": false, "properties": false } }
+    ],
```

Comment:
I had to make this rule a bit more loose after upgrading `"@typescript-eslint/eslint-plugin"`. See https://github.com/typescript-eslint/typescript-eslint/issues/4650 and https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/no-misused-promises.md#checksvoidreturn for context. The rule is not less strict that it was.